src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { Person, PersonRole } from '@hmcts/rpx-xui-common-lib';
+import { select, Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+
+import { PERSON_ERROR_MESSAGE } from '../../../constants';
+import { ExclusionNavigationEvent, ExclusionState, ExclusionStateData } from '../../../models';
+import { ExclusionNavigation } from '../../../models/exclusion-navigation.interface';
+import * as fromFeature from '../../../store';
+
+@Component({
+  selector: 'exui-specific-access-approved',
+  templateUrl: './specific-access-approved.component.html'
+})
+export class SpecificAccessApprovedComponent implements OnInit {
+  public ERROR_MESSAGE = PERSON_ERROR_MESSAGE;
+  @Input() public navEvent: ExclusionNavigation;
```

Comment:
doesn't appear to be in use, can be removed?

Reply:
Please re-review - I've cleaned the unnecessary code from the file that I missed.